corehq/apps/cachehq/cachemodels.py

```diff
@@ -110,11 +110,3 @@ class UserReportsDataSourceCache(GenerationCache):
     views = [
         'userreports/data_sources_by_build_info',
     ]
-
-
-class UserReportsReportConfigCache(GenerationCache):
```

Comment:
and Nick, I think that means there's nothing extra you have to do to keep caching single doc gets. Just check that `ReportConfiguration` subclasses one of those two mixins.